docs/sources/administration/image_rendering.md

```diff
@@ -12,7 +12,7 @@ weight = 8
 
 Grafana supports automatic rendering of panels and dashboards as PNG images. This allows Grafana to automatically generate images of your panels to include in [alert notifications]({{< relref "../alerting/notifications.md" >}}).
 
-When an image is being rendered the PNG-image is temporarily written to the file system. When an image is rendered, the PNG image is temporary written to the `png` folder in the Grafana `data` folder.
+When an image is being rendered, the PNG-image is temporarily written to the file system. When an image is rendered, the PNG image is temporary written to the `png` folder in the Grafana `data` folder.
```

Comment:
```suggestion
When an image is being rendered, the PNG image is temporarily written to the file system. When the image is rendered, the PNG image is temporarily written to the `png` folder in the Grafana `data` folder.
```

Reply:
@oddlittlebird Would it make sense to say "While an image is being rendered" here? The two sentences starting with when read ambiguous to me.